include/lsst/meas/astrom/SipTransform.h

```diff
@@ -0,0 +1,357 @@
+// -*- LSST-C++ -*-
+
+/*
+ * LSST Data Management System
+ * Copyright 2016 LSST/AURA
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_MEAS_ASTROM_SipTransform_INCLUDED
+#define LSST_MEAS_ASTROM_SipTransform_INCLUDED
+
+#include "lsst/afw/geom/LinearTransform.h"
+#include "lsst/meas/astrom/PolynomialTransform.h"
+
+
+namespace lsst {
+
```

Comment:
Better yet, put all `namespace foo { namespace bar { ...` on the same line, as with the other files